PyEveLiveDPS/fleetConnectionWindow.py

```diff
@@ -39,66 +44,110 @@ def __init__(self, mainWindow):
-        tk.Radiobutton(self, text="FC (requires fleet boss)", variable=self.modeVar, value=2).grid(row=self.counter, column="1", sticky="w")
+        tk.Radiobutton(self,
+                       text=tr("FC (requires fleet boss)"),
+                       variable=self.modeVar,
+                       value=2).grid(row=self.counter, column="1", sticky="w")
         self.counter += 1
-        tk.Frame(self, height="10", width="10").grid(row=self.counter, column="1", columnspan="5")
+        tk.Frame(self, height="10", width="10").grid(row=self.counter,
+                                                     column="1",
+                                                     columnspan="5")
         self.counter += 1
-        
+
         self.lowCPUVar = tk.BooleanVar()
         self.lowCPUVar.set(False)
-        lowCPUCheckbutton = tk.Checkbutton(self, text="Use low CPU mode", variable=self.lowCPUVar)
-        lowCPUCheckbutton.grid(row=self.counter, column="1", columnspan="2", sticky="w")
-        descriptor = tk.Label(self, text="Most features will be disabled, but networking will work normally")
+        lowCPUCheckbutton = tk.Checkbutton(self,
+                                           text=tr("Use low CPU mode"),
+                                           variable=self.lowCPUVar)
+        lowCPUCheckbutton.grid(row=self.counter,
+                               column="1",
+                               columnspan="2",
+                               sticky="w")
+        descriptor = tk.Label(
+            self,
+            text=tr(
+                "Most features will be disabled, but networking will work normally"
+            ))
         font = tkFont.Font(font=descriptor['font'])
         font.config(slant='italic')
         descriptor['font'] = font
-        descriptor.grid(row=self.counter+1, column="1", columnspan="2", sticky="w")
-        tk.Frame(self, height="10", width="10").grid(row=self.counter+2, column="1", columnspan="2")
+        descriptor.grid(row=self.counter + 1,
+                        column="1",
+                        columnspan="2",
+                        sticky="w")
+        tk.Frame(self, height="10", width="10").grid(row=self.counter + 2,
+                                                     column="1",
+                                                     columnspan="2")
         self.counter += 3
-        
-        tk.Frame(self, height="10", width="1").grid(row="99", column="1", columnspan="5")
-        
+
+        tk.Frame(self, height="10", width="1").grid(row="99",
+                                                    column="1",
+                                                    columnspan="5")
+
         buttonFrame = tk.Frame(self)
         buttonFrame.grid(row="100", column="0", columnspan="10")
-        self.loginButton = tk.Button(buttonFrame, text="  Login  ", command=self.login)
+        self.loginButton = tk.Button(buttonFrame,
+                                     text="  Login  ",

```

Comment:
This is my own i18n inexperience, but are commands like 'Login', 'Cancel', 'OK', universally understood?

Reply:
Sorry! Looks like they are missed. Gotta check them all again! 😝